integrations/amplitude/HISTORY.md

```diff
@@ -1,3 +1,11 @@
+# 3.4.0 / 2019-10-09
```

Comment:
This is great! Thanks for putting both changes together, it make them easier to deploy. Since we are doing this, we just need to bump the version once, so `v3.3.0` would work here.